source/mainnet/net/resources/glossary.rst

```diff
@@ -465,6 +465,13 @@ Passive delegation
 
 A form of delegation where a delegator's stake is effectively distributed among all baker pools. It is not associated with a specific baker. Delegators earn lower rewards when delegating to passive delegation than when delegating to a specific baker pool. However, passive delegation is not affected by poor performance of a single baker.
 
+.. _glossary-private-keys:
+
+Private keys
```

Comment:
This section sounds correct to me, but it would probably be good to have an extra set of eyes on it. Do you have any comments on this description of private keys, @abizjak?

Reply:
I resolved it this, just so I'm not blocking anything while I'm away the next two days. Daniel will wait for your approval before merging it, Ales.